### PR TITLE
[master < T1082] Jepsen - pause nemesis work until replica registration is done

### PR DESCRIPTION
**Pull request by as51340**
The feature can be considered complete if:
- the first action nemesis does is indeed after the timeout specified.  :white_check_mark: 
- without proper timeout replica gets killed while it was being restored (only it the nemesis action was kill not network partitioning) :white_check_mark: 
- check the test fails when the action kill-node happens during the replica registration :white_check_mark: 

[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [ ] Write a release note here, including added/changed clauses
- [ ] Tag someone from docs team in the comments

### CODE REVIEW

File: tests/jepsen/src/jepsen/memgraph/nemesis.clj [hunk 1]
```diff
@@ -26,17 +26,28 @@
...
-       (gen/phases (gen/sleep 60))))
+  ))
+
+
+(defn full-generator
+  "Construct nemesis generator."
+  [opts]
+  (gen/phases
+   (gen/log "Waiting replicas to get data from main.")
+   (gen/sleep 40)
```
Comment (Darych) [inline tests/jepsen/src/jepsen/memgraph/nemesis.clj:46]: Willl we wait 40 seconds on each phase or just before the 1st phase?

**Review by Darych — APPROVED** [review]

File: tests/jepsen/src/jepsen/memgraph/nemesis.clj (hunk 1 above)
Comment (as51340) [reply tests/jepsen/src/jepsen/memgraph/nemesis.clj:46]: it should be just before the 1st phase

**Comment by as51340** [comment]
I am quite sure that this is good setup of nemesis but then Jepsen tests occasionally fail. Not sure if we should merge then

**Comment by vpavicic** [comment]
@as51340 do we need a RN?

**Comment by gitbuda** [comment]
In this case no because it's all internal stuff + testing, I would skip this one from the changelog perspective

**Pull request merged**